### PR TITLE
Replace occurences of outdated Ext versions.

### PR DESCRIPTION
**Pull request by KaiVolland**

### CODE REVIEW

File: universal-app.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 Follow these instructions to build your first GeoExt 3 application, using Sencha `cmd`. Supported versions:
 
-* Sencha Cmd v6.0.1.76 and v6.0.2.14
-* ExtJS GPL 6.0.0 (ext.version.number=6.0.0.640) and ExtJS GPL 6.0.1 (ext.version.number=6.0.1.250)
-* GeoExt 3 (from 2015-10-03 and from 2016-08-22)
+* Sencha Cmd v6.2.0.103
+* ExtJS GPL 6.2.0 (ext.version.number=6.2.0.???)
```
Comment (chrismayer) [inline universal-app.md]: Why `???`

Comment (KaiVolland) [reply universal-app.md]: sry. that was a placeholder before i left to do something less important... :dancing_women:

Real value should be `6.2.0.981`

Comment (chrismayer) [reply universal-app.md]: Can you adapt it? Then I'll merge this.

Comment (marcjansen) [reply universal-app.md]: @KaiVolland will you update this?

File: ci/create-sencha-environment.sh [hunk 2]
```diff
@@ -31,23 +31,23 @@ mkdir -p $INSTALL_DIR
...
-# 2) Ext JS (v6.0.0.640)
+# 2) Ext JS (v6.2.0.103)
 if [ ! -f "ext-$SENCHA_EXTJS_VERSION-gpl.zip" ]; then
     wget "http://cdn.sencha.com/ext/gpl/ext-$SENCHA_EXTJS_VERSION-gpl.zip"
 fi
 
 # EXTRACT (if needed)
-# 1) Sencha cmd (v6.0.0.202)
+# 1) Sencha cmd (v6.2.0.103)
 if [ ! -f "SenchaCmd-$SENCHA_CMD_VERSION-linux-amd64.sh" ]; then
```
Comment (marcjansen) [inline ci/create-sencha-environment.sh:46]: I think the `*.sh` file is name like this:

`SenchaCmd-6.2.0.103-linux-amd64.sh`

We only have the `6.2.0` part in the script.

I think it'd be good if we introduced

```bash
SENCHA_CMD_VERSION_FULL="6.2.0.103"
```

and changed that line to read

```bash
if [ ! -f "SenchaCmd-$SENCHA_CMD_VERSION_FULL-linux-amd64.sh" ]; then
```

**Comment by marcjansen** [comment]
Any news here, guys?

**Comment by KaiVolland** [comment]
Added the suggested changes.

**Comment by chrismayer** [comment]
Thanks @KaiVolland, merging now...

**Pull request merged**